src/modules/emote_menu/stores/index.js

```diff
@@ -0,0 +1,83 @@
+import SafeEventEmitter from '../../../utils/safe-event-emitter.js';
+import watcher from '../../../watcher.js';
+import emojiCategories from './emoji-categories.js';
+import channelEmotes from '../../emotes/channel-emotes.js';
+import globalEmotes from '../../emotes/global-emotes.js';
+import ffzChannelEmotes from '../../frankerfacez/channel-emotes.js';
+import ffzGlobalEmotes from '../../frankerfacez/global-emotes.js';
+import Icons from '../components/Icons.jsx';
+
+const COLOUMN_COUNT = 7;
+
+function chunkArray(array, size) {
+  if (array.length <= size) {
+    return [array];
+  }
+  return [array.slice(0, size), ...chunkArray(array.slice(size), size)];
+}
+
+class EmoteStore extends SafeEventEmitter {
+  constructor() {
+    super();
+    watcher.on('channel.updated', () => {
+      this.loadProviders();
+      this.loadEmotes();
+    });
+  }
+
+  get totalRows() {
+    return this.rows.length;
+  }
+
+  loadProviders() {
+    this.providers = [
+      {
+        provider: {
+          id: 'bttv',
+          displayName: 'BetterTTV',
+          icon: Icons.THUMBS_UP,
```

Comment:
are these icons are placeholder for actual logos?

Reply:
Yes